### ISSUE TITLE
Provide .tar.gz that does not require Cython to be already installed

### ISSUE THREAD

**Issue by FlorentJeannot** [wzhvictor/libinjection-python#4]
Hello,

The current release requires Cython to be installed first in the venv , which is not ideal in a production environment that requires only one `requirements.txt` with all dependencies to install (since we can't specify a build order, it's very likely to fail).

I would suggest to add a `pyproject.toml` file to this repository containing the following:

```
[build-system]
requires = ["setuptools", "wheel", "Cython"]
build-backend = "setuptools.build_meta"
```

This is taken from the documentation of Cython: https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#distributing-cython-modules

I tried it on my machine, the `.tar.gz` that I built does not fail when I install it without Cython in my venv.

I can provide a PR if you'd like.

Also, if it can help you, that's the steps I've done:

1. In the repo of `libinjection-python`, after adding the `pyproject.toml`:

```
pip install -U pip setuptools
python setup.py sdist
```

2. In another venv I installed the .tar.gz:

```
❯ pip install libinjection-python-1.1.5.tar.gz --no-cache-dir
Processing ./libinjection-python-1.1.5.tar.gz
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: libinjection-python
  Building wheel for libinjection-python (pyproject.toml) ... done
  Created wheel for libinjection-python: filename=libinjection_python-1.1.5-cp39-cp39-macosx_12_0_x86_64.whl size=85957 sha256=fe3f51d869b447227b80dc56d87246f265d4242fc854c873c8cdefd50970f3b5
  Stored in directory: /private/var/folders/4k/bthbhn197slfwjs0z2zgct1h0000gn/T/pip-ephem-wheel-cache-vv_z_l9a/wheels/fe/41/9f/490361f95cd052689d5439209bbdad4c9af9f1aa88682de1fb
Successfully built libinjection-python
Installing collected packages: libinjection-python
Successfully installed libinjection-python-1.1.5
```

3. I ensured that it worked by importing the lib and running the two examples described in the README.

**Comment by FlorentJeannot** [comment]
It also works without `build-backend = "setuptools.build_meta"` in the `pyproject.toml`, which I think is not necessary for this repository.

PEP 517 says:

> If the pyproject.toml file is absent, or the build-backend key is missing, the source tree is not using this specification, and tools should revert to the legacy behaviour of running setup.py (either directly, or by implicitly invoking the setuptools.build_meta:__legacy__ backend).

https://peps.python.org/pep-0517/

**Comment by FlorentJeannot** [comment]
@wzhvictor I added a PR for this if you want to try it out: https://github.com/wzhvictor/libinjection-python/pull/5

**Comment by FlorentJeannot** [comment]
Fixed in #6
